App4/App4/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите значение x для вычисления фнукции y=f(x)");
+        var x = float.Parse(Console.ReadLine());
+        var y = Math.Sqrt(x + 1 / Math.Sqrt(Math.Pow(x, 2) + 4));
```

Comment:
При вычислении квадрата числа проще использовать умножение
```suggestion
        var y = Math.Sqrt(x + 1 / Math.Sqrt(x * x + 4));
```